morpheus/stages/general/monitor_stage.py

```diff
@@ -231,11 +229,15 @@ def sink_on_completed():
                 MorpheusTqdm.monitor.exit()
                 MorpheusTqdm.monitor = None
 
-        stream = seg.make_sink(self.unique_name, self._progress_sink, sink_on_error, sink_on_completed)
+        def node_fn(input: srf.Observable, output: srf.Subscriber):
```

Comment:
nit pick: `observable: srf.Observable, subscriber: srf.Subscriber`

Reply:
So I have changed everything to `obs: srf.Observable, and sub: srf.Subscriber` just to keep it short. Resolving all similar comments.